### PR TITLE
Add tests for module policy

### PR DESCRIPTION
**Pull request by baixinsui**
fixed https://github.com/eclipse-xpanse/xpanse/issues/986

### CODE REVIEW

File: modules/security/src/main/java/org/eclipse/xpanse/modules/security/IdentityProviderManager.java [hunk 1]
```diff
@@ -43,33 +40,8 @@ public void loadActiveIdentityProviderServices() {
...
-                public String getAuthorizeUrl() {
-                    return null;
-                }
-
-                @Override
-                public TokenResponse getAccessToken(String code) {
-                    return null;
-                }
-            };
+            log.error("Not found any identity provider service is active.");
```
Comment (swaroopar) [inline modules/security/src/main/java/org/eclipse/xpanse/modules/security/IdentityProviderManager.java:43]: Log line can be -> No active identity providers found.

File: modules/policy/src/main/java/org/eclipse/xpanse/modules/policy/policyman/PolicyManager.java [hunk 2]
```diff
@@ -211,7 +190,7 @@ public void deletePolicy(UUID id) {
         policyStorage.deletePolicyById(id);
     }
 
-    private void checkPolicyWhetherCreated(Csp csp, String policy) {
+    private void checkPolicyDuplicate(Csp csp, String policy) {
```
Comment (swaroopar) [inline modules/policy/src/main/java/org/eclipse/xpanse/modules/policy/policyman/PolicyManager.java:193]: method name can be - `checkIfPolicyIsDuplicate`

**Review by swaroopar — CHANGES_REQUESTED** [review]
Looks great. Just two minor text changes.

File: modules/policy/src/main/java/org/eclipse/xpanse/modules/policy/policyman/PolicyManager.java (hunk 2 above)
Comment (baixinsui) [reply modules/policy/src/main/java/org/eclipse/xpanse/modules/policy/policyman/PolicyManager.java]: OK

File: modules/security/src/main/java/org/eclipse/xpanse/modules/security/IdentityProviderManager.java (hunk 1 above)
Comment (baixinsui) [reply modules/security/src/main/java/org/eclipse/xpanse/modules/security/IdentityProviderManager.java]: OK

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**